8_bit_pic.py

```diff
@@ -339,7 +370,7 @@ def nible_in_bytes(self) -> int:
 
     def fetch(self):
         next_ip = self.pc + self.word_in_bytes
-        self.ic = int.from_bytes(self.RAM[self.pc: next_ip], "big")
+        self.ic = int.from_bytes(self.RAM[self.pc : next_ip], "big")
```

Comment:
I did not realize that PEP8 calls for this - https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements

flake8 disagrees:
![image](https://github.com/user-attachments/assets/7ddfa116-dd91-4e69-a05e-831279587a8f)

Reply:
I still write it [something: otherthing] every time and have the formatter fix it and I write python everyday 🤣 ... but also how often do you really pass in both indexes if you think about it lol